guides/release/components/template-tag-format.md

```diff
@@ -0,0 +1,285 @@
+The template tag format is a powerful, new way to write components in Ember. It's a single-file format that combines the component's JavaScript and Glimmer template code. The `<template>` tag is used to keep a clear separation between the template language and the JavaScript around it.
```

Comment:
In also unifies around the concept of ES Modules, which old components did not. Idk if that's too much no mention is the opening paragraph tho

Reply:
I fear that goes a bit too deep. It's super interesting to know and realize, but it also requires you to have prior understanding of what the concept of ES Modules are. And I don't want people not being familiar with that being scared off.

A separate section/article about the more technical concepts or reasoning behind the syntax would be more suiting. But I think that could be a separate PR.